lineage/Analyze.py

```diff
@@ -96,7 +96,7 @@ def run_Analyze_over(list_of_populations, num_states, parallel=True, **kwargs):
     return output
 
 
-def Results(tHMMobj, pred_states_by_lineage, LL):
+def Results(tHMMobj, pred_states_by_lineage, LL, fpi, fT, fE):
```

Comment:
The Results function should know if we have fixed parameters that were given in analysis.

Reply:
Outdated, ignore.